openfl/utilities/utils.py

```diff
@@ -236,3 +236,22 @@ def merge_configs(
             settings.set(key, operation(value))
     settings.validators.validate()
     return settings
+
+
+def change_tags(tags, add_field=None, remove_field=None) -> Tuple[str]:
+    """Change tensor tags to add or remove fields.
+
+    Args:
+        tags(tuple): tensor tags.
+        add_field(str): add a new tensor tag field.
+        remove_field(str): remove a tensor tag field.
+    """
+    tags = list(tags)
+
+    if add_field:
+        tags.append(add_field)
```

Comment:
May be we should remove duplicates after adding field too? 
Beacause I expect that 
`change_tags(('a', 'b'), 'c', 'c') == change_tags(('a', 'b', 'c'), 'c', 'c')` 
but 
```python
change_tags(('a', 'b', 'c'), 'c', 'c') == ('a', 'b', 'c')
change_tags(('a', 'b'), 'c', 'c')  == ('a', 'b')
```